### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by mpobaschnig**
Currently draft until documentation has been fully reviewed.

### CODE REVIEW

File: docs/01_SDKIntroduction/SDKIntroduction.rst [hunk 1]
```diff
@@ -69,7 +69,7 @@ Supported Operating Systems
 Following operating systems are supported by Oxygen:
 
  * Microsoft Windows 7,10,11 (64bit)
- * Ubuntu 20.04 LTS Linux (Focal Fossa)
+ * Ubuntu 20.04 LTS Linux (Focal Fossa) and 22.04 LTS Linux (Jammy Jallyfish)
```
Comment (matthiasstraka) [inline docs/01_SDKIntroduction/SDKIntroduction.rst:72]: Jellyfish

File: docs/02_Quickstart/Quickstart.rst [hunk 2]
```diff
@@ -79,7 +79,7 @@ In order to build and execute unit tests, `Boost C++ Libraries 1.70 <https://www
 Development Requirements for Linux
 ----------------------------------
 
-- Ubuntu 20.04 LTS Linux (Focal Fossa)
+- Ubuntu 20.04 LTS Linux (Focal Fossa) or Ubuntu 22.04 LTS Linux (Jammy Jallyfish)
```
Comment (matthiasstraka) [inline docs/02_Quickstart/Quickstart.rst:82]: Jellyfish

**Review by matthiasstraka — CHANGES_REQUESTED** [review]

File: docs/01_SDKIntroduction/SDKIntroduction.rst (hunk 1 above)
Comment (mpobaschnig) [reply docs/01_SDKIntroduction/SDKIntroduction.rst]: Done.

File: docs/02_Quickstart/Quickstart.rst (hunk 2 above)
Comment (mpobaschnig) [reply docs/02_Quickstart/Quickstart.rst]: Done.

**Review by matthiasstraka — APPROVED** [review]

**Review by matthiasstraka — APPROVED** [review]

**Pull request merged**